lib/bungee/src/Output.cpp

```
@@ -0,0 +1,76 @@
+	else
+	{
+		lappedSynthesisBuffer.unpadded().topRows(3 * lappedSynthesisBuffer.frameCount) = lappedSynthesisBuffer.unpadded().middleRows(lappedSynthesisBuffer.frameCount, 3 * lappedSynthesisBuffer.frameCount);
+		lappedSynthesisBuffer.unpadded().middleRows(3 * lappedSynthesisBuffer.frameCount, lappedSynthesisBuffer.frameCount).setZero();
+	}
+}
+
+inline auto makeOutputChunk(Eigen::Ref<Eigen::ArrayXXf> ref)
+{
+	OutputChunk outputChunk{};
+	outputChunk.data = ref.data();
+	outputChunk.frameCount = (int)ref.rows();
```

Cppcheck (2.19.0)

[style] The function 'toRadians' is never used.

(unusedFunction)